scripts/superdesk-archive/related-item-widget/relatedItem.js

```diff
@@ -56,12 +56,45 @@
                         $scope.options.item.slugline = item.slugline;
                         $scope.options.item.related_to = item._id;
                         api.save('archive', $scope.origItem, $scope.options.item).then(function(_item) {
-                            notify.success(gettext('item updated.'));
+                            notify.success(gettext('item metadata associated.'));
                             return item;
                         });
                     },
                     'class': 'open',
-                    icon: 'icon-expand'
+                    icon: 'icon-expand',
+                    condition: function(item) {
+                        return true;
+                    }
+                },
+                update: {
+                    title: 'Associate as update',
+                    method: function(item) {
+
+                        api.save('archive_rewrite', {},
+                            {'update': angular.extend({}, $scope.origItem, $scope.item)},
+                            item)
+                        .then(function(new_item) {
+                                    notify.success(gettext('Story is associated as update.'));
+                                    authoringWorkspace.edit(new_item._id);
+                                }, function(response) {
+                                    if (angular.isDefined(response.data._message)) {
+                                        notify.error(gettext('Failed to associate update: ' + response.data._message));
+                                    } else {
+                                        notify.error(gettext('There is an error. Failed to associate update.'));
+                                    }
+                                });
+                    },
+                    'class': 'open',
+                    icon: 'icon-expand',
+                    condition: function(item) {
+                        var destinationRewrite = !_.contains(['published', 'corrected', 'killed', 'spiked'], $scope.item.state) &&
```

Comment:
Spiked items don't appear in the related item widget and use the `authoring.isPublished()` function.